nextflow_schema.json

```diff
@@ -842,11 +842,22 @@
                 },
                 "busco_db": {
                     "type": "string",
-                    "description": "Download URL for BUSCO lineage dataset, or path to a tar.gz archive, or local directory containing already downloaded and unpacked lineage datasets.",
-                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz or '/path/to/buscodb' (files still need to be unpacked manually). Available databases are listed here: https://busco-data.ezlab.org/v5/data/lineages/.",
+                    "description": "Download URL, tar.gz archive local path, or local uncompressed directory for an *_odb10 or *_odb12 BUSCO lineage datasets.",
+                    "help_text": "E.g. https://busco-data.ezlab.org/v5/data/lineages/bacteria_odb10.2024-01-08.tar.gz, `/path/to/bacteria_odb10.2024-01-08.tar.gz`.\n\nFor more details, check the [BUSCO section in the usage documentation](https://nf-co.re/mag/latest/docs/usage/#busco).",
                     "format": "path",
                     "exists": true
                 },
+                "busco_db_lineage": {
+                    "type": "string",
+                    "description": "Name of the BUSCO *_odb10 or *_odb12 lineage to check against.",
+                    "help_text": "Typically used in combination with `--busco_db`, corresponding to the lineage files supplied within the `--busco_db` file or directroy.\n\nIf supplying without a `--busco_db`, or not supplying this `--busco_db_lineage` nor `--busco_db` may require an internet connection for BUSCO's autodetection to work!",
+                    "pattern": ".*_odb(10|12)$"
+                },
+                "busco_force_offline": {
+                    "type": "boolean",
+                    "description": "Force BUSCO to run in offline mode.",
+                    "help_text": "This will prevent BUSCO from trying to download lineage datasets from the internet, and instead use the local files specified in `--busco_db`, ensuring reproducibility."
```

Comment:
```suggestion
                    "help_text": "This will prevent BUSCO from trying to download lineage datasets from the internet, and instead use the local files specified in `--busco_db`, ensuring reproducibility. Ensure you have sufficient lineages in your `--busco_db` pre-downloaded and unpacked"
```